CONTRIBUTING.md

```diff
@@ -1,55 +1,48 @@
-big issues into smaller pieces of work for people to bite off. We don't expect
-contributors to complete all the work before submitting their first patch.
-Instead, we prefer Contributors to take little bites and submit small patches so
-that we can develop tight, asynchronous feedback loops.
-
-We use labels like `design`, `documentation`, `test`, `code`, `infrastructure`,
-and `security` to indicate what kind of work is remaining on the issue.
+New contributors may want to review
+[the Zinc contribution guide](https://www.zinc.coop/contributing/) to ground themselves in the norms we try to follow across projects.
 
-We tend to “merge quickly and often” so long as the patch is safe to apply. If
-you prefer your patch not get merged, please mark the
-[PR as a draft](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests).
+### 1.1 General Design and Architecture.
+[`Convene`] is as a multi-user, multi-application web-first Operating System.
 
-New contributors may want to review
-[the Zinc contribution guide](https://www.zinc.coop/contributing/) for an
-overview of the norms we follow across projects.
+Convene's design strategy leans heavily into [Outcome-Driven Innovation](https://en.wikipedia.org/wiki/Outcome-Driven_Innovation), [Accessibility](https://developer.mozilla.org/en-US/docs/Web/Accessibility), [Usability](https://en.wikipedia.org/wiki/Usability), and [Data Portability](https://en.wikipedia.org/wiki/Data_portability).
+Interoperability is offered by embracing open protocols like [Hypermedia as the Engine of Application State (HATEOAS)](https://en.wikipedia.org/wiki/HATEOAS). Quality and Changeability is maintained by performing [Test-Oriented Development](https://en.wikipedia.org/wiki/Test-driven_development), [Continuous Deployment](https://en.wikipedia.org/wiki/Continuous_deployment), and [Domain Driven Design](https://en.wikipedia.org/wiki/Domain-driven_design).
 
-### 1.1. Architecture
 
-At present, Convene is split into three modules:
+### 1.2 Users and Stakeholders.
+Stakeholders for `Convene` fall into one or more of the following categories:
 
-The root directory provides the human and programmatic interface for Convene
-as well as data-persistence; with two additional submodules.
+- [`Clients`], who are the legal owners of the data of their [`Spaces`] in a [`Neighborhood`].
+- [`Members`], have broad authority over [`Spaces`] and their [`Furniture`]. They are also responsible for [`Visitor`: Support], [`Visitor`: Trust & Safety], [`Member`: Support], and [`Member`: Trust & Safety].
+- [`Visitors`], who access [`Spaces`] and their [`Furniture`] according to [`Member`] specification.
+- [`Operators`], who steward the [`Neighborhood`: Infrastructure], [`Neighborhood`: Trust & Safety], and [`Client`: Support] responsibilities
+- [`Contributors`] and [`Maintainers`], who design and build [`Furniture`].
```

Comment:
So, are these that don't link to anything yet meant to be linked later?

Reply:
I am also happy to kick this back to drafts and ensure all `[` have links before merging.

Reply:
I am of the mindset that as long as that change is understood to exist, then this should be good to go in its present state.